### PR TITLE
Hoping to make it easier to understand this page helps with Taps

### PR DESCRIPTION
**Pull request by sethladd**
Not everyone will understand that gesture == tap.

### CODE REVIEW

**Comment by sethladd** [comment]
cc @abarth

File: gestures.md [hunk 1]
```diff
@@ -10,9 +10,13 @@ permalink: /gestures/
 
 # Introduction
 
+This document explains how to listen for, and respond to,
+touch, taps, and gestures in Flutter. Collectively, touch, taps,
+and gestures are considered part of the _gesture system_.
```
Comment (abarth) [inline gestures.md]: Those three things aren't parallel...  Maybe "taps, drags, and other gestures"?

Comment (sethladd) [inline gestures.md]: Love it. Can do!

**Comment by sethladd** [comment]
@abarth ptal. Thanks!

**Comment by abarth** [comment]
LGTM

**Comment by sethladd** [comment]
Thanks!

Can land this once https://github.com/flutter/flutter/issues/8910 is addressed.

**Comment by sethladd** [comment]
Going to merge, because the site is broken now anyway (this PR didn't introduce newly broken links). These builds should go green once https://github.com/flutter/website/issues/447 lands.

**Pull request merged**